internal/node/node.go

```diff
@@ -117,6 +117,11 @@ type NodeClient struct {
 
 type args map[string]interface{}
 
+const (
+	defaultTimeout  = 10 * time.Second
+	extendedTimeout = 4 * time.Minute
```

Comment:
What does this mean?

Reply:
after testing, the longer timeout is apparently not needed. i tried to deploy a deployment with 10 vms using the default timeout (10 secs) and it worked fine. because the node client only makes calls to nodes, and the nodes respond immediately after receiving them, without waiting for workloads to be StateOK, this happens somewhere else in the deployer.
i update the calls to use the default timeout